torch/csrc/jit/script/builtin_functions.cpp

```diff
@@ -33,6 +33,11 @@ def warn(string: str):
   print(string)
 )SCRIPT";
 
+auto python_builtins_source_overloads = R"SCRIPT(
+def warn(string: str, stacklevel: int):
```

Comment:
Can you just add stacklevel as a default arg ? Or is that not yet implemented here

Reply:
Not in the Python string frontend so leaving it as an overload for now